doc/datasets/index.rst

```diff
@@ -254,6 +254,36 @@ features::
 
  _`Faster API-compatible implementation`: https://github.com/mblondel/svmlight-loader
 
+.. _external_datasets:
+
+Loading from external datasets
+==============================
+
+scikit-learn works on any numeric data stored as a numpy array or a pandas DataFrame, 
```

Comment:
really, we support a numpy array or scipy sparse matrix. But other types that are convertible to numeric arrays are acceptable.